graphql/src/store/query.rs

```diff
@@ -53,15 +53,57 @@ pub(crate) fn build_query<'a>(
         query = query.filter(filter);
     }
     let order = match (
-        build_order_by(entity, field)?,
+        build_order_by(entity, field, schema)?,
         build_order_direction(field)?,
     ) {
-        (Some((attr, value_type)), OrderDirection::Ascending) => {
+        (Some((attr, value_type, None)), OrderDirection::Ascending) => {
             EntityOrder::Ascending(attr, value_type)
         }
-        (Some((attr, value_type)), OrderDirection::Descending) => {
+        (Some((attr, value_type, None)), OrderDirection::Descending) => {
             EntityOrder::Descending(attr, value_type)
         }
+        (Some((attr, _, Some(child))), OrderDirection::Ascending) => match child {
```

Comment:
Put a check just before the match that if `ENV_VARS.graphql.disable_child_sorting` is set, this returns `QueryExecutionError::NotSupported("Sorting by child attributes is not supported")`

Reply:
I did :)